drivers/wddr/driver.c

```diff
@@ -82,6 +82,15 @@ void wddr_set_chip_select_reg_if(wddr_dev_t *wddr, wddr_channel_t channel, wddr_
     wddr_set_channel_top_cfg_reg_if(wddr, channel, ca_reg_val, dq_reg_val);
 }
 
+void wddr_read_bscan_result_reg_if(uint8_t *result, wddr_dq_byte_t dq_byte, wddr_channel_t channel)
```

Comment:
Let's pass in wddr device as a parameter like other functions. Be sure to use the base address. Move result to the end or you can just make it return a uint8_t value instead of void.

Reply:
Fixed.